src/lib.rs

```diff
@@ -48,33 +53,72 @@ pub fn headtail(opts: &Opts) -> Result<()> {
                     }
                 }
             }
-            Err(e) => return Err(e),
         }
     }
 
     // Keep following(?)
-
+    //
+    // To avoid wasted CPU cycles, we can use a file system watcher (e.g.
+    // `inotify(7)` on Linux).
+    //
+    // The `notify` crate provides several optimized file system watchers using
+    // functionality built into operating systems. Should an optimized watcher
+    // not be available, `notify` will default to a polling watcher.
```

Comment:
This information needs to be included in some (probably shorter) form in the `help` documentation for the `-f/--follow` option.

Reply:
✅